<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_addBoxesAutomaticDistribution_selectMethod</class>
 <widget class="QDialog" name="Dialog_addBoxesAutomaticDistribution_selectMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add boxes using automatic distribution</string>
  </property>
  <widget class="QPushButton" name="pushButton_closeAsPossible">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Boxes as close to each other as possible</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectDistanceBetweenBoxesCenter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set distance between box centers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the method to use:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
